src/EngineManager.php

```diff
@@ -76,10 +107,10 @@ public function createNullDriver()
      */
     public function getDefaultDriver()
     {
-        if (is_null($this->app['config']['scout.driver'])) {
+        if (is_null($this->container['config']['scout.driver'])) {
```

Comment:
Because we are using `config()` on the 75, maybe we can use  `config()` here too?